bin/index.sh

```diff
@@ -27,4 +27,4 @@ npm run wp-env run cli wp theme activate pub/wporg-learn-2024
 npm run wp-env run cli wp rewrite structure '/%postname%/'
 
 # Import content
-# npm run wp-env run cli "php bin/import-test-content.php"
+# npm run wp-env run cli php bin/import-test-content.php
```

Comment:
This line is commented out. When I uncomment it the import works now without the quotes. Did you intend to comment it?

Reply:
I thought that it was intentionally left uncommented. I made the changes to the build process and uncommented the import command.

I'm new to the project, so I'm unsure, but it looks like the build didn't work as expected.

The courses have been imported to the local database, I can see Tutorials and Lesson Plans, but the front end looks like this:

localhost:8888
<img width="1726" alt="Screenshot 2024-06-06 at 7 26 46 AM" src="https://github.com/WordPress/Learn/assets/808069/2cd40906-f72d-4932-8b09-53e77a4c06cd">

localhost:8888/tutorials
<img width="1727" alt="Screenshot 2024-06-06 at 7 47 49 AM" src="https://github.com/WordPress/Learn/assets/808069/428ec589-13a2-4928-a16f-ade90a24956f">

I suppose it should match: https://learn.wordpress.org/tutorials/?

I also looked at the [build workflow](https://github.com/WordPress/Learn/blob/trunk/.github/workflows/build.yml) and tried it out but had no luck. Any ideas?

Reply:
No that's pretty much as expected. The theme is very much a work in progress; the 'No learning pathways found' message needs aligning, and there will be no Tutorials moving forward, so that is the default archive template you're seeing there. We can update the content import at some point to drop the out of date content and add some Lessons and Courses.

Thanks again for your help, I'll do a final test and get this merged.